orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -509,8 +509,13 @@ def run_step(model, x):
         assert torch.allclose(ort_prediction, pt_prediction)
         _test_helpers.assert_gradients_match_and_reset_gradient(ort_model, pt_model)
 
-@pytest.mark.parametrize("use_fp16", [False, True])
-def test_gradient_correctness_conv1d(use_fp16):
+@pytest.mark.parametrize("use_fp16, input_requires_grad", [
```

Comment:
@pytest.mark.parametrize("use_fp16", [...])
@pytest.mark.parametrize("input_requires_grad", [...])

I saw somewhere this would also work.